spec/schema.json

```diff
@@ -25,7 +25,8 @@
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string"
+              "type": "string",
+              "pattern": "^[A-Za-z]{2}[0-9]{6}$"
```

Comment:
That would exclude ordinary company numbers. I appreciate that we are only dealing with OE numbers for now but if it's not OE-specific it should probably allow any valid company number

Reply:
Altered this to accept all company numbers.